libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -1852,7 +1852,7 @@ def write_UART_config(self, f):
             f.write('#define HAL_SERIAL%s_DRIVER Empty::UARTDriver serial%sDriver\n' %
                     (idx, idx))
 
-        if 'IOMCU_UART' in self.config:
+        if 'IOMCU_UART' in self.config and False:
```

Comment:
Wouldn't this just make it never be true?

Reply:
> Wouldn't this just make it never be true?

Yep!  IOMCU is currently not working :-)  Like a lot of things ;-)

Reply:
I do not get the point of this. I will ask you about this on a Dev meeting.

Reply:
> I meant, wouldn't this break other vehicles too?

Oh yes, very thoroughly.  This is an interim commit while I work out what to do about IOMCU.  It wasn't a problem on boards other than Pixhawk6X that I was using, because no IOMCU.

Need to be able to be able to make a derivative board from a hwdef, removing the IOMCU.  That's not really possible at the moment because IOMCU is a basic directive you can put into a hwdef and can't undo by something which is including that hwdef.  Possibly by making `AP_IOMCU_ENABLED` actually work....  Don't know, I was in hack-and-slash mode for that bit as I wanted to get a hardware debugger in there, and had Pixhawk6X on my desk for that :-)

Reply:
I've now found a better way to get this effect, so removed this hack.  It did its job :-)